### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by brunokiafuka**
Done some minor changes to the source code: 

- [x] Added dev dependencies 
- [x] Updated readme 
- [x] Added type to path node obj. I.e: ` ReturnStatement: (path: object) => {}`

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -15,38 +15,37 @@ add(2, 3);
...
+  Identifier: (path: object) => {
+    if (path.isIdentifier({ name: "$log" })) {
       hasLogID = true;
       path.remove();
     }
   },
-  BinaryExpression: (path) => {
-    if(path.node.operator === '+') {
-      path.node.operator = '*'
+  BinaryExpression: (path: object) => {
```
Comment (Fausto95) [inline index.ts]: You can remove this actually, it was just for testing purpose

File: index.ts [hunk 2]
```diff
@@ -15,38 +15,37 @@ add(2, 3);
...
     path.traverse(LoggerVisitor);
-  }
+  },
 });
 
 const output = generator(ast, {}, code);
 const newCode = output.code;
-console.log({newCode: eval(newCode)})
\ No newline at end of file
+console.log({ newCode: eval(newCode) });
```
Comment (Fausto95) [inline index.ts]: We can also remove this

File: index.ts [hunk 3]
```diff
@@ -15,38 +15,37 @@ add(2, 3);
...
       path.replaceWithMultiple([
         t.identifier(`console.log('Final Result 😛 ==> ', ${identifierName})`),
-        t.identifier(`return ${identifierName}`)
+        t.identifier(`return ${identifierName}`),
       ]);
     }
   },
-  Identifier: (path) => {
-    if(path.isIdentifier({name: '$log'})) {
+  Identifier: (path: object) => {
```
Comment (Fausto95) [inline index.ts]: typing it as `object` makes it very weak
We should install `@types/babel-traverse` and import the Path type declaration

Comment (brunokiafuka) [reply index.ts:30]: will be using `@types/babel__traverse` instead. It seems to be up to date.

Comment (brunokiafuka) [reply index.ts]: I am using the `@babel/types` so the line will plus minus look like this`Identifier: (path: t.Node) => {...}`. 

`t.Node` will replace the object is the preferred type to the path as it receives a set of different interfaces that can attend to the different use cases.

Comment (Fausto95) [reply index.ts]: Nice, go for it

**Review by Fausto95 — APPROVED** [review]

**Pull request merged**